include/opentype-sanitiser.h

```diff
@@ -58,6 +58,8 @@ class OTSStream {
 
   virtual ~OTSStream() {}
 
+  virtual size_t SizeLimit() = 0;
```

Comment:
I’d rename it `size()`.

Reply:
Done. Existing methods in OTSStream seemed to use CamelCase but I see that the trival one chksum() does not.